build.gradle.kts

```diff
@@ -28,14 +28,22 @@ import org.jetbrains.gradle.ext.settings
 plugins {
     // Apply third-party plugins.
     alias(libs.plugins.dependencyAnalysis)
+    alias(libs.plugins.gitSemver)
     alias(libs.plugins.ideaExt)
     alias(libs.plugins.versions)
 }
 
+semver {
+    // Do not create an empty release commit when running the "releaseVersion" task.
+    createReleaseCommit = false
+
+    // Do not let untracked files bump the version or add a "-SNAPSHOT" suffix.
+    noDirtyCheck = true
+}
+
 // Only override a default version (which usually is "unspecified"), but not a custom version.
 if (version == Project.DEFAULT_VERSION) {
-    val gitVersionProvider = providers.of(GitVersionValueSource::class) { parameters { workingDir = rootDir } }
-    version = gitVersionProvider.get()
+    version = semver.version
```

Comment:
Could you summarize the algorithm to calculate the version here (and maybe also in the commit message)?

Reply:
It's in the README, just [scroll down a bit](https://github.com/jmongard/Git.SemVersioning.Gradle?tab=readme-ov-file#versioning) from the link provided in the commit message.

Reply:
I've also added a brief description to the commit message.